src/main/resources/db/migration/postgres/V1_333_1__insert_security_groups.sql

```diff
@@ -0,0 +1,33 @@
+DO $$
+DECLARE
+	recordToBeMigrated record;
+	newId integer;
+	oldId integer;
+BEGIN
+FOR recordToBeMigrated IN
+    SELECT grp_id from security_group where grp_id >0
+    order by 1
+	LOOP
+	select recordToBeMigrated.grp_id+1000000 into newId;
```

Comment:
Its probably never likely to be a problem but it may be worth making the number negative rather than positive. The sequence number we use always increments so by inserting negative numbers we are guranteed to never clash

Reply:
i've been asked to leave the negative numbers alone, hence the `and sg1.grp_id > 0` clause in src/main/resources/db/migration/postgres/V1_333_3__insert_security_groups.sql

Remember, these scripts are onyl going to be run against the test databases now, so unless there is data right now that is over 1000000, then it wont matter.